go/tools/builders/asm.go

```diff
@@ -94,6 +94,12 @@ func buildSymabisFile(goenv *env, sFiles, hFiles []fileInfo, asmhdr string) (str
 			seenHdrDirs[hdrDir] = true
 		}
 	}
+	// The package path has to be specified as of Go 1.22 or the resulting
+	// object will be unlinkable, but the -p flag is only required in
+	// preparing symabis since Go1.22
+	if packagePath != "" && isGo119OrHigher() {
```

Comment:
```suggestion
	if packagePath != "" && isGo121OrHigher() {
```
I think you meant to do this?

Reply:
Ah yeah, thanks